### PR TITLE
Add ReadCube link to publication

### PR DESCRIPTION
**Pull request by drpatelh**
Many thanks for contributing to nf-core/tools!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/tools/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -563,9 +563,9 @@ INFO: Successfully synchronised [n] pipelines
...
 
-> **The nf-core framework for community-curated bioinformatics pipelines**
+> **The nf-core framework for community-curated bioinformatics pipelines.**
 >
-> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
 >
-> [_Nat Biotechnology_ (2020)](https://doi.org/10.1038/s41587-020-0439-x);
-> doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x)
+> _Nat Biotechnol._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).
```
Comment (ewels) [inline README.md]: Do we need the short version? I think the long version is prettier 🙄 

```suggestion
> _Nature Biotechnology_ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).  
```

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/README.md [hunk 2]
```diff
@@ -69,9 +69,9 @@ For further information or help, don't hesitate to get in touch on [Slack](https
...
 
-> **The nf-core framework for community-curated bioinformatics pipelines**
+> **The nf-core framework for community-curated bioinformatics pipelines.**
 >
-> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
 >
-> [_Nat Biotechnology_ (2020)](https://doi.org/10.1038/s41587-020-0439-x);
-> doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x)
+> _Nat Biotechnol._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).
```
Comment (ewels) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/README.md]:
```suggestion
> _Nature Biotechnology._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).  
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,7 +27,7 @@
 ### Other
 
 * Improve documentation for installing `nf-core/tools`
-* Add details of the new nf-core publication in Nature Biotechnolgy :champagne:
+* Add details of the new nf-core publication in Nature Biotechnology :champagne:
```
[the remark this answers is not in the document]
Comment (drpatelh) [reply CHANGELOG.md]: 😆

File: README.md (hunk 1 above)
Comment (drpatelh) [reply README.md]: I quite like the short version and thats how its always listed on PubMed. Not fussed either way to be honest. Shall I change it back?

**Comment by drpatelh** [comment]
> Looks good, but maybe we can put this on hold whilst we look in to making the manuscript open access?
Yep. No worries. I wanted to get this in the next release just in case we forget. 

> It's a bit more long-lasting in the template than it would be on the website.. Though I guess it doesn't hurt to have the readcube link either way.
Exactly. No harm done if its there anyway?

File: README.md (hunk 1 above)
Comment (ewels) [reply README.md]: I guess it’s just a personal preference thing. Having a combination of short and long form as I wrote is definitely wrong though!

**Pull request merged**